pywr/_core.pyx

```diff
@@ -843,11 +807,11 @@ cdef class Storage(AbstractStorage):
         AbstractStorage.setup(self, model)
         # Setup any Parameters and Recorders
         if self._cost_param is not None:
-            self._cost_param.setup(model)
+            self._cost_param.setup()
```

Comment:
I think this is no longer needed. The parameter.setup method will be called by model directly.